src/App.tsx

```diff
@@ -48,6 +48,20 @@ function App() {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
+  // pulls from ISO api to get standardized country names, codes and translations
+  useEffect(() => {    
+    const updateCountriesJson = async () => {
+      const api = new httpClient()
+      // TODO: Figure out a better place to put this so we don't keep updating this either 
```

Comment:
Is this comment also true here? I've seen this comment alot and it might just be an artifact of copy paste

Reply:
It -technically- makes sense, but I don't know if we would ever want to store the JSON data

Reply:
I think that comment was an artifact from a copy paste -removed it.